### ISSUE TITLE
Duplicate lines in panel DXF

### ISSUE THREAD

**Issue by elcojacobs** [Rat-Rig/V-core-3#33]
The DXF files for the panels seem to have some issues.

- Almost all lines are duplicated.
- The file has about 20 layers, but they seem empty except for one
- Not all lines are joined well.

When I use the DXF file to get a quote, the quote is 25 euros higher compared to a cleaned-up file.

First I fixed the files by deleting all the extra stuff in Inkscape, but a good DXF can actually be exported form the 3D model on Onshape directly.
Open the linked document for the panels.
Right-click a panel and choose 'export DXF'.

I don't know if there is a good reason for the current DXFs to be like they are. But it would be easy to export the panels from Onshape and commit them to the repo instead of the current files.

**Comment by pkucmus** [comment]
Hi, thanks for letting us know. I assume you checked the metric ones only? Or maybe all of them?

Onshape has a nasty bug of exporting every DXF in inches instead of the set document unit. That combined with how DXF is designed results in exports that are 450 inch where they should be 450mm (quite drastic). Plenty of software asks about the unit on import and then all is fine but a lot of shops around the world were defaulting to inch (or whatever the reason there was) and people were reporting that those are impossible to cut. 
It's a constant battle with those.

**Comment by elcojacobs** [comment]
That doesn't seem to happen when I export them. But that's probably because my default units in Onshape are set to metric?
I attached a file that I exported from Onshape. I had to rename it for github.
[panes-left-onshape.dxf.txt](https://github.com/Rat-Rig/V-core-3/files/7072663/panes-left-onshape.dxf.txt)

**Comment by pkucmus** [comment]
So apparently you exported in the 1998 version of DXF (`AC1014` in line 8). As far as I understand DXF your `INSUNITS` is referencing a object `100000B7` (line 4228) and that object defines `1` which is Inch in line 6594

from https://images.autodesk.com/adsk/files/autocad_2012_pdf_dxf-reference_enu.pdf page 20:
![image](https://user-images.githubusercontent.com/6347418/131257092-eb810c03-f0ee-4375-9d35-48fd4f02650c.png)

The V-Core 3 document is set to milimeters so is my workspace but still Onshape returns `1` under `$INSUNITS` which tricks many readers into thinking it's in inches. Inkscape apparently ignores `$INSUNITS`, same goes for FreeCAD but apparently it's not the case for all of the readers (as users reported)

Inkscape on a direct export from Onshape:
![image](https://user-images.githubusercontent.com/6347418/131257191-dc6e604d-293c-4c61-a9d6-636e10781bca.png)

**Comment by elcojacobs** [comment]
All I can tell you is that the DXF's that are provided by RatRig cause problems with automated systems at shops that cut sheets due to duplicate lines. Tried them at both kunststofplatenshop.nl and 247tailorsteel.

The DXF created by right-clicking a panel on Onshape and choosing 'DXF/DWG export' creates no problems.
Onshape DXF (from panel, not from DXF in tree):
![image](https://user-images.githubusercontent.com/1708921/131262898-911650e9-fe36-4369-b266-249ab98f03ef.png)
File attached, just to be sure I gave you the right one last time.
[400-left.dxf.txt](https://github.com/Rat-Rig/V-core-3/files/7073108/400-left.dxf.txt)


Github DXF:
![image](https://user-images.githubusercontent.com/1708921/131262985-207659ec-1e9a-4960-ac9e-b1fffe81d8fb.png)
